core/crypto/murmur/murmur.cpp

```diff
@@ -64,7 +64,7 @@ namespace fc::crypto::murmur {
     }
 
     // tail
-    auto tail = gsl::span(input).subspan(blocks * kBlockSize);
+    auto tail = input.subspan(blocks * kBlockSize);
```

Comment:
I would specify subspan size.

Reply:
I would not, because it's "tail" and because it will duplicate offset (`input.size() - blocks * kBlockSize`)